Manager/src/test/java/io/raspberrywallet/manager/ManagerTest.java

```diff
@@ -78,4 +81,14 @@ void getAvailableBalance() {
         assert availableBalance.equals(mockAvailableBalance);
     }
 
+    @Test
+    void getCpuTemperature() {
+        final String mockCpuTemp = "75 °C";
```

Comment:
milicelsius: aka 75000

Reply:
This is what `Manager` should return after formatting.